CONTRIBUTING.md

```diff
@@ -1,26 +1,82 @@
+ 
+Workflow
+------------------------------
+ 
+1. Create a branch directly in this repo or a fork (if you don't have push access). Please name
+branches within this repository `feature/<description>` or `fix/description`. For example, 
+something like `feature/upgrade_agent_0.2.3-70`.
 
-1. Create a branch directly in this repo or a fork (if you don't have push access). Please name branches within this repository `feature/<description>` or `fix/description`. For example, something like `feature/upgrade_agent_0.2.3-70`.
+2. Create an issue or open a pull request (PR). If you aren't sure your PR will solve the issue
+or may be controversial, we're okay with you opening an issue separately and linking to it in 
+your PR. That way, if the PR is not accepted, the issue will remain and be tracked.
 
-1. Create an issue or open a PR. If you aren't sure your PR will solve the issue, or may be controversial, we commend opening an issue separately and linking to it in your PR, so that if the PR is not accepted, the issue will remain and be tracked.
+3. Clone the fork/branch locally:
 
-1.  Close (and reference) issues by the `closes #XXX` or `fixes #XXX` notation in the commit message. Please use a descriptive, useful commit message that could be used to understand why a particular change was made.
+    
+        $ git clone https://github.com/Netuitive/chef-netuitive.git
+    
+4. Close (and reference) issues by the `closes #XXX` or `fixes #XXX` notation in the commit
+message. Please use a descriptive, useful commit message that could be used to understand why a
+particular change was made.  <b>Example:</b>
 
-1. Keep pushing commits to the initial branch, using `--amend`/`--rebase` as  necessary. Please don't mix fixing unrelated issues in a single branch.
+        
+        $ git add *.txt
+        $ git commit -m "Fixes #xxx. Here's a useful commit message about why the change was made."
+        $ git push origin name-of-fix-or-feature
+        
+5. Keep pushing commits to the initial branch using `--amend`/`--rebase` as necessary. Don't mix 
+unrelated issues in a single branch.
 
-1. When everything is ready for merge, clean up the branch (rebase with master to synchronize, squash, edit commits, etc) to prepare for it to be merged.
+6. Clean up the branch (rebase with master to synchronize, squash, edit commits, test, etc.) to
+prepare for it to be merged.
 
-### Merging contributions
+7. If you didn't open a pull request already, do so now. Your pull request should have the following:
+    * Descriptive commit messages.
+    * Updated documentation.
+    * Passed continuous integration (CI) tests
+    * Version bumps
+    * A changelog
+    
+8. After further review, a project maintainer can merge your request.
 
-1. After reviewing commits for documentation, passing CI tests, version bumps, changelog, and good descriptive commit messages, a maintainer can merge.
-
-
-### Releasing
```

Comment:
I reworked this into a different bullet, but the original bullet says it better. I'll change what I have.